doc/ansible/dev/testing_guide.md

```diff
@@ -7,7 +7,7 @@ To begin, you sould have:
 - The latest version of the [operator-sdk](https://github.com/operator-framework/operator-sdk) installed.
 - Docker installed and running
 - [Molecule](https://github.com/ansible/molecule) >= v2.20
-- [Ansible](https://github.com/ansible/ansible) >= v2.7
+- [Ansible](https://github.com/ansible/ansible) >= v3.6
```

Comment:
Ansible does not have a v3.6. This requirement can stay v2.7, since all anisble versions >= v2.5 support python 3

Reply:
It is true. Thank you for the good spot.